docs/docsource/installation/upgrade/upgrade_xdm.rst

```diff
@@ -7,22 +7,22 @@ Upgrade XDM
    
    .. parsed-literal::
       
-      curl -u '$CREDS' https://eve.devsca.com/github/scality/zenko/artifacts/builds/github:scality:zenko:staging-|version|.r210210230152.7c4bbdc.pre-merge.00015938/zenko-|version|-beta.3.iso -o zenko-|version|-beta.3.3.iso
+      curl -u '$CREDS' https://eve.devsca.com/github/scality/zenko/artifacts/builds/github:scality:zenko:staging-|version|.r210210230152.7c4bbdc.pre-merge.00015938/zenko-|version|.iso -o zenko-|version|.iso
```

Comment:
I think you're going to have to get an engineer to spell out what "r210210230152.7c4bbdc.pre-merge.00015938" is. This may be constrained to the version (like a checksum, for example) and may need to be updated in our documentation with each version (in which case, a substitution variable may be in order). It may be tied to the individual installation, in which case, we need to describe what this number/hash is, and where the reader can find and copy it.

Reply:
Yes indeed, do you know the meaning of this number @xaf-scality ? Here, |version| = 2.0.0.

Reply:
r210210230152.7c4bbdc.pre-merge.00015938 is a specific build id, my approximate guess is:
r: revision?
210210230152: timestamp probably UTC: 21-02-10 23:01:52
7c4bbdc: Most likely the git hash of the commit
pre-merge: eve build step name, we use pre-merge a lot
00015938: I believe this is the build number for eve

This changes with every eve build and tbh I wouldn't include anything that is not promoted in any of our docs as they expire after two weeks. The promoted ones have a release version followed by `-promoted`.

Reply:
We'll need to spell out how the reader will access these hashes.

Reply:
Maybe @tcarmet  knows more but I wouldn't expect eve to be our preferred source of artifacts.

Reply:
Yes, it seems it is an internal link.

Reply:
This is a packaging failure. There is no .iso in the public GitHub repository, and Eve is not public. Where are users going to find the .iso file? In earlier versions, the only engineer who had an opinion insisted that the customers build their own .iso file, and that if they didn't have that skill, they weren't clever enough to use our product. I did not find this answer satisfactory, but I do not recall how we resolved the issue. 

Please keep pushing on both development and release engineering until we get a meaningful solution.

Reply:
How will the end user acquire the .iso file?

Reply:
For open source user I believe the argument is that they should build it themself, and for customers it's going to be hosted on packages.scality.com